salvo/src/lib/benchmark/fully_dockerized_benchmark.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module contains the methods to perform a nighthawk benchmark using
+containers for the scripts, nighthawk binaries, and envoy
+
+https://github.com/envoyproxy/nighthawk/blob/master/benchmarks/README.md
+"""
+
+import logging
+
+from src.lib.benchmark.base_benchmark import BaseBenchmark
+
+log = logging.getLogger(__name__)
+
+
+class Benchmark(BaseBenchmark):
+
+  def __init__(self, job_control, benchmark_name, **kwargs):
```

Comment:
Looks like the comment about `kwargs` was only partially addressed. We do have exact arguments now, but we kept the `kwargs` also.

Does this mean we don't know the complete list of arguments here? And if that is the case, can we discuss this and figure out an architecture where the full list of arguments passed through the layers is known?

If we do know the complete list of arguments, can we remove the `kwargs`?

As you have pointed out - this applies to a few locations in this PR.

Reply:
I'll remove it here since it's unused.  The other places pass kwargs to things like `subprocess.check_output` and the method to run a docker image.  In those cases kwargs seems much more compact than having a long list of parameters and values.